JSAsynchronous/js/script.js

```diff
@@ -0,0 +1,20 @@
+const promiseArray = capitalize(['music', 'javascript', 'cinema', 'php']);
+
+function capitalize(arrWord) {
+    return new Promise(function(resolve, reject){
+        const uppercaseArray = arrWord.map(function(item) { 
+            if(typeof item === "string") {
+                return item.toUpperCase();
+            }else {
+                reject(new Error("array include elements different from string"));
+            }            
+        });
+        resolve(uppercaseArray);
+    });
+}
+
+promiseArray
+    .then(function(arrWord){
+        console.log(arrWord.sort());
```

Comment:
Use sortWords function as it is mentioned in the Word document! The task is related to promises so use them (your solution is fine but we want to practice promises here :) )

Reply:
I added sortWords a function and realized asynchrony.

Reply:
Can you implement it similar to the other functionality - use promise